### ISSUE TITLE
Dropdown inside dropdown issue

### ISSUE THREAD

**Issue by asafge** [mjolnic/bootstrap-colorpicker#57]
I've used bootstrap-colorpicker inside another bootstrap dropdown, and it looks like it's not behaving properly. Specifically - when double clicking on the round selector it will close the top dropdown (and leave the color picker open). Same behavior when clicking in a "white" area outside the colors div.

What I've done is added this code near the end of the ```var Colorpicker``` function/constructor definition:
```
        this.picker.on({
            'click': function(e) {
                e.stopPropagation();
                e.preventDefault();
            },
            'dblclick': $.proxy(this.hide, this)
        });
```

So basically click won't propagate upwards and thus the upper drop-down will not close.
Also (and I know this is not the current behavior), I've added a double-click handler that closes the picker. This is just a suggestion though.

The reason I'm not sending this out as a pull request is that I'm not sure what is the maintainers' desired behavior in this case, so I'd love to hear what you guys think.

Thanks

**Comment by mjolnic** [comment]
@asafge Hi, any updates on this? Could you provide an example in JSFiddle to reproduce this issue ? Thanks
